pkgs/stdenv/generic/setup.sh

```diff
@@ -1527,6 +1517,7 @@ showPhaseFooter() {
     local startTime="$2"
     local endTime="$3"
     local delta=$(( endTime - startTime ))
+    echo "finished phase: $phase"
```

Comment:
this is a dupe of the `completed in` no?

Reply:
If we're adding a print like this always then it makes sense to add the completed in to it. The reason I limited it to >30 is because i wanted to avoid the extra line